src/test/java/uk/gov/justice/hmpps/casenotes/controllers/CaseNoteResourceTest.kt

```diff
@@ -422,6 +422,190 @@ class CaseNoteResourceTest : ResourceTest() {
         .expectStatus().isBadRequest
   }
 
+  @Test
+  fun testCanSoftDeleteCaseNote() {
+    oAuthApi.subGetUserDetails("DELETE_CASE_NOTE_USER")
+    elite2Api.subGetOffender("A1234BA")
+    val token = jwtHelper.createJwt("DELETE_CASE_NOTE_USER", roles = DELETE_CASENOTE_ROLES)
+
+    val postResponse = webTestClient.post().uri("/case-notes/{offenderIdentifier}", "A1234BA")
+        .headers(addBearerToken(token))
+        .bodyValue(CREATE_CASE_NOTE_WITHOUT_LOC.format("This is a case note"))
+        .exchange()
+        .expectStatus().isCreated
+        .returnResult(CaseNote::class.java)
+    val id = postResponse.responseBody.blockFirst()!!.caseNoteId
+
+    webTestClient.get().uri("/case-notes/{offenderIdentifier}/{caseNoteIdentifier}", "A1234BA", id)
+        .headers(addBearerToken(token))
+        .exchange()
+        .expectStatus().isOk
+        .expectBody()
+        .json(readFile("A1234BA-single-casenote.json"))
+
+    webTestClient.delete().uri("/case-notes/{offenderIdentifier}/{caseNoteId}", "A1234BA", id)
+        .headers(addBearerToken(token))
+        .exchange()
+        .expectStatus().isOk
+
+    val notFoundBody = String.format("{\"status\":404,\"developerMessage\":\"Resource with id [%s] not found.\"}", id);
```

Comment:
this is kotlin you're writing - you can do variable interpolation directly in the strings.  Worth doing a `"""` string and then you don't need to escape the `"` too